tests/batching/test_adaptive.py

```diff
@@ -11,6 +11,9 @@
 
 from .conftest import TestRequestSender
 
+# from ..metrics.conftest import prometheus_registry
+from prometheus_client.registry import CollectorRegistry
```

Comment:
Just as a last thing, after moving the fixture to `conftest.py`, these imports are not required anymore (linter will complain otherwise)